docs/source/index.rst

```diff
@@ -64,7 +64,7 @@ Maintainers
 
 - Manuella Dias Cruz: https://github.com/ManuDiasCruz
 - Jonathan Carvalho: https://github.com/johnrc90
-
+- Guilherme Cardoso: https://github.com/gkardoso
```

Comment:
I think you can also adds your name here @henrique-lh

Reply:
I'm not sure, I just changed the docs haha